common.mk

```diff
@@ -257,6 +261,12 @@ BUILDROOT_TOOLCHAIN=toolchain-aarch$(COMPILE_NS_USER)
 else
 BUILDROOT_TOOLCHAIN=toolchain-aarch$(COMPILE_NS_USER)-legacy
 endif
+else ifeq ($(UNAME_M),aarch64)
```

Comment:
Is this some new syntax in GNU make or has this always worked?

Reply:
As per [docs](https://www.gnu.org/software/make/manual/html_node/Conditional-Syntax.html#Conditional-Syntax), it looks like this has always worked.

Reply:
Nice, I've totally missed that. :-)

Reply:
No worries :).